### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by braingram**

### CODE REVIEW

**Review by zacharyburnett — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -14,23 +14,21 @@ for reading and writing chunked
 Installation
 ------------
 
-This plugin is not yet stable and released on
-PyPi and requires features only available in the
-current development head of ASDF.
+This extension is available on PyPi. We are actively developing
```
Comment (zacharyburnett) [inline README.rst:17]:
```suggestion
This extension is available on PyPI. We are actively developing
```

File: README.rst [hunk 2]
```diff
@@ -14,23 +14,21 @@ for reading and writing chunked
...
+this extension and until a stable release (1.0.0) is made it
```
Comment (zacharyburnett) [inline README.rst:18]:
```suggestion
this extension and until a stable release (`1.0.0`) is made it
```

File: README.rst [hunk 3]
```diff
@@ -14,23 +14,21 @@ for reading and writing chunked
...
+is possible that breaking changes will be introduced. If you
+are using this extension please let us know so we can look at
```
Comment (zacharyburnett) [inline README.rst:20]:
```suggestion
are using this extension please let us know, so we can look into
```

**Comment by zacharyburnett** [comment]
sorry, just saw some minor tweaks I didn't see on first glance!

**Comment by braingram** [comment]
No worries. I will open up a follow up. Thanks for taking a look!

**Pull request merged**